setup.py

```diff
@@ -6,6 +6,7 @@
 from os import path
 
 here = path.abspath(path.dirname(__file__))
+completion_dir = '/'.join([path.expanduser('~'), 'bash_completion.d/'])
```

Comment:
You may want to use `os.path.join` instead of `"/".join`, because on some operating systems "\" should be used instead.  :(